examples/OECC/config/response_agent_1.3.0.properties

```diff
@@ -0,0 +1,14 @@
+#Install Configurations
+#----------------------
+USER_INSTALL_DIR=/usr/oecc_agent
+OECC_AGENT_AS_SERVICE=1
+OECC_JAVA_HOME=/usr/lib/jvm/jdk
```

Comment:
Is this JDK location consistent for Ubuntu installations?

Reply:
The JDK is installed using the script common/scripts/install_openjdk.sh.
It is configured to install to that location.
Added a note on this to the README.md.